src/kata5.reachDestination.js

```diff
@@ -1,3 +1,8 @@
-const reachDestination = (distance, speed) => {};
+const reachDestination = (distance, speed) => {
+    let journeyTime = distance / speed; 
+   let  journeyTime2 = Math.round(journeyTime * 2) / 2 
```

Comment:
Same here, this can be a constant. I would also rename this one to something like roundedJourneyTime to be more descriptive of the difference between the two variables.